x-pack/plugins/apm/common/environment_filter_values.ts

```diff
@@ -33,3 +33,18 @@ export const ENVIRONMENT_NOT_DEFINED = {
 export function getEnvironmentLabel(environment: string) {
   return environmentLabels[environment] || environment;
 }
+
+export function getEnvironmentUrlParam(
+  requestedEnvironment?: string,
+  queryEnvironment?: string
+) {
+  const normalizedRequestedEnvironment =
+    requestedEnvironment || ENVIRONMENT_NOT_DEFINED.value;
+  const normalizedQueryEnvironment = queryEnvironment || ENVIRONMENT_ALL.value;
```

Comment:
I'm wondering if we should change the default so that if no environment is set (in the url) it'll default to `ENVIRONMENT_NOT_DEFINED.value`.

Ideally when empty I think it should pick "production" if that's available or the top environment. This is similar to what we do with transaction type.

Reply:
Hmm, I think all is a good default here - there's no all option for transaction types so maybe not the right thing to compare it to.